tests/tools/kopiarunner/kopiarun.go

```diff
@@ -73,8 +72,31 @@ func (kr *Runner) Run(args ...string) (stdout, stderr string, err error) {
 	c.Stderr = errOut
 
 	o, err := c.Output()
-
 	log.Printf("finished '%s %v' with err=%v and output:\nSTDOUT:\n%v\nSTDERR:\n%v", kr.Exe, argsStr, err, string(o), errOut.String())
 
 	return string(o), errOut.String(), err
 }
+
+// RunAsync will execute the kopia command with the given args in background
+func (kr *Runner) RunAsync(args ...string) (*exec.Cmd, error) {
+	argsStr := strings.Join(args, " ")
+	log.Printf("running '%s %v'", kr.Exe, argsStr)
+	// nolint:gosec
+	cmdArgs := append(append([]string(nil), kr.fixedArgs...), args...)
+	// nolint:gosec
+	c := exec.Command(kr.Exe, cmdArgs...)
+	c.Env = append(os.Environ(), kr.environment...)
+
+	errOut := &bytes.Buffer{}
+	c.Stderr = errOut
+	err := c.Start()
+
+	if err != nil {
+		return nil, err
+	}
+	if _, errOutput := c.Output(); errOutput != nil {
```

Comment:
I think this is the opposite of what you want. `c.Output()` calls `c.Run()`, which waits for the command to complete before returning its output. Not sure what the behavior is if you call `c.Start()`, then `c.Run()`. Can you verify this change is working, at least by running the engine against a local filesystem repo? LMK if you have already done so, or if you run into any problems setting it up.

Reply:
Reading the exec package code, I expect `errOutput` would be set to `"exec: already started"`, and we take the error path.

Reply:
@redgoat650 I was not able to see this error string `"exec: already started"` in the logs, doesn't seem to wait for the output else tests would not complete, and I tried running full robustness-suite.

Reply:
Seems like I/m not returning the `errOutput` let me try to run this again

Reply:
Thanks for figuring this out @redgoat650 , made the change pushing commit.